man/QDNAseq-package.Rd

```diff
@@ -32,10 +32,13 @@
 
 \section{How to cite this package}{
     Whenever using this package, please cite:
-    Scheinin I, Sie D, Bengtsson H, van de Wiel MA, Olshen AB, van Thuijl HF, van Essen HF, Eijk PP, Rustenburg F, Meijer GA,
-Reijneveld JC, Wesseling P, Pinkel D, Albertson DG and Ylstra B (2014). "DNA copy number analysis of fresh and
-formalin-fixed specimens by shallow whole-genome sequencing with identification and exclusion of problematic regions in the
-genome assembly." _Genome Research_, *24*, pp. 2022-2032.
+    Scheinin I, Sie D, Bengtsson H, van de Wiel MA, Olshen AB, van Thuijl
+HF, van Essen HF, Eijk PP, Rustenburg F, Meijer GA, Reijneveld JC,
+Wesseling P, Pinkel D, Albertson DG and Ylstra B (2014). "DNA copy
+number analysis of fresh and formalin-fixed specimens by shallow
+whole-genome sequencing with identification and exclusion of
+problematic regions in the genome assembly." _Genome Research_, *24*,
+pp. 2022-2032.
```

Comment:
OK, thanks. I don't think this is worth spending time on, was just curious. (My setup is R 3.2.3 on OS X.)